README.md

```diff
@@ -242,15 +242,35 @@ sudo apt-get -y install git cmake python-dev build-essential automake autoconf l
 
 2. use the `vsbuild.sh` script in the `sh` directory.
 
-or manually
+OR
+
+2a. Configure and compile VS manually using the ncurses ccmake frontend:
```

Comment:
This will not render correctly. You can look at the preview of Markdown file to see how it looks.
A minor thing, so I won't block on this.
![image](https://user-images.githubusercontent.com/38861757/81194837-caaebd00-8fc5-11ea-9ebd-03ae5dddfaf5.png)